tools/tune/eq/eq_define_parametric_eq.m

```diff
@@ -271,20 +271,20 @@
 	% Don't let Q go negative, which causes an unstable filter.
 	Q = max(0.0, Q);
 
-	if frequency <= 0 || frequency >= 1
-		% When frequency is 0 or 1, the z-transform is 1.
-		b = [1, 0, 0];
-		a = [1, 0, 0];
-		return;
-	endif
-	if Q <= 0
-		% When Q = 0, the above formulas have problems. If we
-		% look at the z-transform, we can see that the limit
-		% as Q->0 is 0, so set the filter that way.
-		b = [0, 0, 0];
-		a = [1, 0, 0];
-		return;
-	endif
+    if frequency <= 0 || frequency >= 1
+        % When frequency is 0 or 1, the z-transform is 1.
+        b = [1, 0, 0];
+        a = [1, 0, 0];
+        return;
+    end
+    if Q <= 0
+        % When Q = 0, the above formulas have problems. If we
+        % look at the z-transform, we can see that the limit
+        % as Q->0 is 0, so set the filter that way.
+        b = [0, 0, 0];
+        a = [1, 0, 0];
+        return;
+    end
```

Comment:
cleanup, not compatibility. Also no point to change indent by 8 to 4. Since FW code uses 8 with tab, we could default to it for Matlab. The indents are well visible with 8, and a huge accumulated indent indicates issue in code.

Reply:
Sure, I will change my editor indent settings with 8. Thanks  !!